content/about/notes/1.1/index.md

```diff
@@ -2,9 +2,12 @@
 title: Istio 1.1
 weight: 90
```

Comment:
is this the correct weight?  I believe you want 89.

Reply:
this is done